docs/upgrade/upgrade-security.asciidoc

```diff
@@ -13,35 +13,23 @@ NOTE: For detailed information about the 8.0 release, check out the <<release-no
 [[upgrade-reqs]]
 == Upgrade requirements
 
-*Track rules that are automatically disabled when upgrading*
-
-Upon upgrading to 8.0, rules are automatically disabled. Once the upgrade completes, you should re-enable them again to avoid gaps in rule coverage. _We highly recommend that you track your active rules before upgrading so you can easily find and re-enable them after upgrading to 8.0_.
-
-Before upgrading, use the <<rules-api-find, Find rules API>> to capture a list of enabled rules.
+*Upgrade {agent}s*
 
-IMPORTANT: Use curl or another HTTP tool to run {elastic-sec} API commands.
+Upgrade your {stack} and {agent}s to 7.17 first (refer to {fleet-guide}/upgrade-elastic-agent.html[Upgrade Fleet-managed Elastic Agents]). After this is complete, you can  {stack-ref}/upgrading-elastic-stack.html[upgrade the {stack}] to 8.0. Initially, {agent}s will be version 7.17; this is fine because {elastic-sec} supports the last minor release in 7.x (7.17) and any subsequent {elastic-endpoint} versions in 8.x. After the {stack} upgrade, you can decide whether to upgrade {agent}s to 8.0, which is recommended to ensure you get the latest features.
 
-Here is a curl command that you can run to find enabled rules:
+NOTE: You do not need to shut down your {agent}s or endpoints to upgrade the {stack}.
 
-[source,console]
---------------------------------------------------
-GET /api/detection_engine/rules/_find?per_page=10000&filter=alert.attributes.enabled:true
---------------------------------------------------
+*Re-enable disabled rules*
 
-To re-enable your rules from the Rules page:
+Any active rules prior to upgrading from 7.17 will be automatically disabled while upgrading to 8.0, and a tag named `auto_disabled_8.0` will be added to those rules for tracking purposes. Once the upgrade is complete, you can filter rules by the newly added tag, then use bulk actions to re-enable them:
```

Comment:
I know this might look a bit out of place since the rest of the topic describes pre/post-req steps for upgrading to 8.0, but should the first sentence in this section describe what happens to active rules when upgrading from 7.x to 8.0.1 and above? The automatic process of disabling and tagging previously enabled rules was introduced in 8.0.1 so unless that change was backported to 8.0, users will need to install 8.0.1 (not 8.0) to have their rules handled this way, right?